{{ cookiecutter.project_name }}/pyproject.toml

```diff
@@ -18,12 +18,12 @@ keywords = []
 requires-python = ">=3.12"
 dependencies = []
 
+[project.optional-dependencies]
+test = ["pytest>=8.4.2", "pytest-cov>=6.3.0", "pytest-sugar>=1.1.1"]
+
 [project.urls]
 homepage = "https://github.com/nens/{{ cookiecutter.project_name }}"
 
-[tool.pytest.ini_options]
-addopts = "--cov --cov-fail-under=80"
-
```

Comment:
Good one. I had forgotten that it breaks vscode.
A bummer, as this setting is a nice way to customise the way`pytest` runs in the github action without having to make changes to the generated gh action file.